### PR TITLE
Out null deployment

### LINKED ISSUE #1028: Suppress return value output for New-TestResources.ps1

### PR DESCRIPTION
**Pull request by danieljurek**
Fixes #1028 

Example lacking Out-Null (scroll to the bottom of the logs) -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=523905&view=logs&j=b28a1a35-334b-5681-3876-e5562b5e8062&t=ff48ff62-22a8-5437-aae5-18a32a8994ec&l=237

Example with Out-Null (scroll to the bottom of the logs) -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=543132&view=logs&j=b28a1a35-334b-5681-3876-e5562b5e8062&t=85d3da62-e026-56d3-c19c-0f96227fb514&l=160

### CODE REVIEW

File: tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        public WeeklyIntegrationTestingPipelineConvention(ILogger logger, PipelineGenerationContext context) : base(logger, context)
+        {
+        }
+
+        protected override string GetDefinitionName(SdkComponent component)
+        {
+            return $"{Context.Prefix} - {component.Name} - tests-weekly";
+        }
+
+        protected override Schedule CreateScheduleFromDefinition(BuildDefinition definition)
```
Comment (mitchdenny) [inline tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs:17]: This change seems to be in two PRs?

**Review by heaths — APPROVED** [review]
Script changes LGTM, though assigning to `$null` is faster when performance matters more (just a tip).

File: tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs (hunk 1 above)
Comment (weshaggard) [reply tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs:17]: Agreed lets separate these changes into their own PR.

Comment (weshaggard) [reply tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs:17]: Looks like the pipeline generation changes are already part of https://github.com/Azure/azure-sdk-tools/pull/1026

**Review by mitchdenny — DISMISSED** [review]
Pipeline Generator changes need to be moved out (you've got it covered in another PR).

**Comment by weshaggard** [comment]
I updated this PR to remove the pipeline generation changes.

**Pull request merged**